lib/cocoapods/target/framework.rb

```diff
@@ -0,0 +1,30 @@
+module Pod
+  class Framework
```

Comment:
I'm terrible at naming things - let me know if you have a better name idea

Reply:
`FrameworkPath`?